lib/perfdata/opentsdbwriter.cpp

```diff
@@ -210,6 +210,14 @@ void OpenTsdbWriter::CheckResultHandler(const Checkable::Ptr& checkable, const C
 					
 					continue;
 				}
+
+				if (value.IsEmpty()) {
+					Log(LogDebug, "OpenTsdbWriter")
+						<< "Resolved macro '" << missing_macro
+						<< "' for this host or service to '', skipping.";
```

Comment:
I think you can enhance the log message a bit (currently it states "...for this host or service", but it's not clear which service it refers to).
```bash
[2024-02-21 17:43:30 +0100] debug/OpenTsdbWriter: Resolved macro '' for this host or service to '', skipping.
```

Reply:
Something like this would be nicer:
```bash
[2024-02-21 17:43:30 +0100] debug/OpenTsdbWriter: Resolved macro '' for checkable 'my!checkable' to '', skipping.
```